Autocoders/Python/src/fprime_ac/generators/visitors/CommandVisitor.py

```diff
@@ -89,7 +89,7 @@ def _writeTmpl(self, c, fp, visit_str):
     def DictStartVisit(self, obj):
         """
         Defined to generate files for generated code products.
-        @parms obj: the instance of the command model to visit.
+        @params obj: the instance of the command model to visit.
```

Comment:
These should be param. This flag is pulled in by sphinx autodocs.

Reply:
Perhaps this is a change we should make after this PR merely given the size.

Reply:
It's not a big deal for me to apply corrections, especially since I'll have to make a couple anyway. (And rebase.)

Fwiw, I'll probably perform that in two passes: one as a strict "drop/correct" pass, and another as a "rebase" pass -- with a comment between, to make it possible to try to review the shift between each pass.

That said, I don't think I'll make a push until you finish your second pass.